tenant-isolation-config-service-api/src/main/proto/org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_config_service.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package org.hypertrace.tenantisolation.config.service.v1;
+
+option java_multiple_files = true;
+
+import "org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_group_config.proto";
+
+service TenantIsolationConfigService{
+  rpc GetAllTenantIsolationGroupConfigs(GetAllTenantIsolationGroupConfigsRequest) returns (GetAllTenantIsolationGroupConfigsResponse) {}
+  rpc CreateTenantIsolationGroupConfig(CreateTenantIsolationGroupConfigRequest) returns (CreateTenantIsolationGroupConfigResponse) {}
+  rpc UpdateTenantIsolationGroupConfig(UpdateTenantIsolationGroupConfigRequest) returns (UpdateTenantIsolationGroupConfigResponse) {}
+  rpc DeleteTenantIsolationGroupConfig(DeleteTenantIsolationGroupConfigRequest) returns (DeleteTenantIsolationGroupConfigResponse) {}
+}
+
+message GetAllTenantIsolationGroupConfigsRequest{
```

Comment:
check style - should be a space before brace at least

Reply:
fixed